lib/wanda_web/views/v1/execution_json.ex

```diff
@@ -70,6 +63,12 @@ defmodule WandaWeb.V1.ExecutionView do
 
   defp adapt_v1_agent_check_results(agent_check_results), do: agent_check_results
 
+  defp execution(%Execution{} = execution) do
+    execution
+    |> ExecutionJSON.execution()
```

Comment:
I think it is better to alias `V2` and have here `V2.ExecutionJSON.execution()` it is much more obvious what's going on

Reply:
Fair point :+1: